src/golang/lib/models/shared/types.go

```diff
@@ -0,0 +1,25 @@
+package shared
```

Comment:
Not sure when to put in shared vs primitives package.

Reply:
I think `utils/primitive.go` should be restricted to only primitive types and NULL primitive types, e.g. `NullInt`, `UUIDSlice`, etc.

Anything else should go in shared.